services/venice-server/src/main/java/com/linkedin/venice/stats/ServerReadQuotaUsageStats.java

```diff
@@ -31,7 +30,7 @@ public class ServerReadQuotaUsageStats extends AbstractVeniceStats {
   private final Sensor rejectedKPS; // rejected key per second
   private final Sensor allowedUnintentionallyKPS; // allowed KPS unintentionally due to error or insufficient info
   private final Sensor usageRatioSensor; // requested kps divided by nodes quota responsibility
-  private final Int2ObjectMap<ServerReadQuotaVersionedStats> versionedStats = new Int2ObjectOpenHashMap<>();
+  private final ConcurrentHashMap<Integer, ServerReadQuotaVersionedStats> versionedStats = new ConcurrentHashMap<>();
```

Comment:
Let's use `VeniceConcurrentHashMap` because of issues with CHM in JDK8 (the javadoc in `VeniceConcurrentHashMap` has the details of the issue)